### DISCUSSION TITLE
Nested dates fields in CRUD Service configuration

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#150] [Q&A]
Hi!

I have a data model that includes a nested date field. So, in MongoDB I would like to have something like the following:
```
{
    "_id" : ObjectId("5d31ccf8a677000013b83de9"),
    "stringField" : "blabla",
    "objField": {
        "dateField": ISODate("2019-07-19T14:00:24.771Z"),
    }
}
```
However, in the console, for nested objects I can only define a JSON Schema that does not support "Date" types.
Is there an alternative way to reach my goal of having nested date types?

Thanks & Regards.

**Comment by fredmaggiowski** [comment]
As of now you could save numeric timestamps instead of  iso dates, I think this requires an implementation @nicocampagna FYI

**Comment by danibix95** [comment]
Currently the service does not support casting dates in nested fields. This feature is supported only at the first level of the document.
This request is tracked in the following issue https://github.com/mia-platform/crud-service/issues/283.
